ts/screens/onboarding/EmailInsertScreen.tsx

```diff
@@ -117,35 +137,81 @@ class EmailInsertScreen extends React.PureComponent<Props, State> {
     });
   }
 
+  private handleGoBack() {
+    if (this.isFromProfileSection) {
+      this.props.navigation.goBack();
+    } else {
+      Alert.alert(
+        I18n.t("onboarding.alert.title"),
+        I18n.t("onboarding.alert.description"),
+        [
+          {
+            text: I18n.t("global.buttons.cancel"),
+            style: "cancel"
+          },
+          {
+            text: I18n.t("global.buttons.exit"),
+            style: "default",
+            onPress: () => this.props.dispatch(abortOnboarding())
+          }
+        ]
+      );
+    }
+  }
+
+  private isFromProfileSection =
```

Comment:
```suggestion
get isFromProfileSection() {
    return this.props.navigation.getParam("isFromProfileSection") || false;
  }
```
A better way to write a getter

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1363/commits/72d80b8de3378b713a0eb6c11d19690aad60667b